SU-HSDO/suhumsci
# HSD8-658 Fix icon hovering over branding bar

# READY FOR REVIEW

# Summary
- Fix search icon when lockup doesnt use all lines. The icon hovers over the branding bar at the top.

# Needed By (Date)
- soon

# Urgency
- low

# Steps to Test
1. Checkout develop to verify issue
1. sync heidi-williams `blt drupal:sync --site=heidi_williams__humsci --partial`
1. view the home page. verify the search icon hovers over brand bar.
 ![search](https://user-images.githubusercontent.com/7185045/59542072-f2ca9480-8eb8-11e9-9ab7-8190fb20d97e.png)
1. checkout this branch
1. reload home page (clear browser caches maybe)
1. verify search icon no longer hovers over brand bar. 
1. verify search still functions normally.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
